libexec/remake-api-migrations.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# -f so that it's OK when there are no migrations to start with
+rm -rf CreeDictionary/API/migrations/*.py
```

Comment:
```suggestion
rm -f CreeDictionary/API/migrations/*.py
```

No need for `-r`ecursive! 😱